docusaurus/docs/reference/ha/bootstrapping.md

```diff
@@ -0,0 +1,164 @@
+---
+sidebar_label: Bootstrapping
+sidebar_position: 10
+---
+
+# Bootstrapping A Cluster 
+
+To bring up a controller cluster, one starts with a single node. 
+
+## Controller Configuration
+
+### Certificates
+
+Each controller requires appropriate certificates. The certificates for clustered controllers 
+have more requirements than those for a standalone server. See the [Certificates Reference](./certificates.md)
+for more information. 
+
+### Config File
+
+The controller requires a `raft` section.
+
+```yaml
+raft:
+  dataDir: /path/to/data/dir
+```
+The `dataDir` will be used to store the following:
+
+* `ctrl-ha.db` - the ziti model bbolt database
+* `raft.db` - the raft bbolt database
+* `snapshots/` - a directory to store raft snapshots
+
+Controller use the control channel listener to communicate with each other. Unlike
+routers, they need to know how to reach each other, so an advertise address must
+be configured.
+
+```yaml
+ctrl:
+  listener: tls:0.0.0.0:6262
+  options:
+    advertiseAddress: tls:192.168.1.100:6262
+```
+ 
+Finally, for sessions to work across controllers, JWTs are used. To enable these
```

Comment:
> for sessions to work across controllers, JWTs are used.

Is this relevant information at this point? It feels like this only makes me ask more questions and makes me want more information. I feel like this should be a "do this" section: "enable the OIDC endpoint" and then you don't have to answer the question about 'why' -- well... we'll see i guess? :)

Reply:
Tweaked this a bit, to make it less thought provoking, maybe better?